packages/builders/src/interactions/slashCommands/mixins/ApplicationCommandStringOptionMinMaxLengthMixin.ts

```diff
@@ -0,0 +1,18 @@
+export abstract class ApplicationCommandStringOptionMinMaxLengthMixin {
```

Comment:
Can't you just re-use the number/integer one but renamed so its more common?

Reply:
numeric one is for value and this one is for length.

Reply:
Oh woops, didn't read the property right